tests/robot-tests/tests/libs/slack.py

```diff
@@ -17,14 +15,10 @@
 class SlackService:
     def __init__(self):
         self.slack_bot_token = os.getenv("SLACK_BOT_TOKEN")
-        self.report_webhook_url = os.getenv("SLACK_TEST_REPORT_WEBHOOK_URL")
```

Comment:
Looks like this variable can be removed after this has been deployed? If you haven't already, I'd consider noting this down along with any other post-deploy tasks.

Reply:
I think removing `EES UI tests` from Slack apps is another post-deploy task.

Reply:
Both on my list 👍